src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -67,12 +79,48 @@ class ActivityFeed extends React.Component<Props, State> {
         if (hasLoaded || hasMoreItems || hasNewItems || hasInputOpened) {
             this.resetFeedScroll();
         }
+
+        if (currCurrentUser !== prevCurrentUser || currFeedItems !== prevFeedItems) {
```

Comment:
We do a check inside of the soon to be renamed `checkAndLogRenderMetrics()` to see if the event has already been logged, to prevent redundant logs. I'll set this up so that it runs that portion of the if check upon currFeedItems being set, and prev being null (as simple as `(currFeedItems && !prevFeedItems)`)

Reply:
Actually, I can just used the above created `hasLoaded`